agent/tests/workflows/test_post_twitter_workflow.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+from unittest.mock import Mock, patch
+from src.workflows.post_twitter.workflow import PostTwitterWorkflow, TweetContent
+
+
+class TestPostTwitterWorkflow:
```

Comment:
We always prefer method pytest's over class-based pytests. Take a look at `agent/tests/workflows/test_triage_message_workflow.py` as an example.

We'll need to use mock out `tweepy` in order to get the test to pass